src/locales/en/translation.json

```diff
@@ -90,6 +90,8 @@
     "meals": "meals"
   },
   "purchase": {
+    "footer": "Certain Gift-a-Meal campaigns will be conducted in partnership with Apex for Youth, a 501(c)(3) nonprofit organization. If a campaign is denoted as such, contributions will be tax deductible.",
```

Comment:
Apologies for the one-more-thing, but would you mind adding these (in english) to the cn translations file? Otherwise they won't even show up when viewing in chinese :)

Reply:
Ah, good catch. My bad.